apps/remix-ide-e2e/src/tests/pluginManager.spec.ts

```diff
@@ -143,7 +154,8 @@ module.exports = {
           .waitForElementVisible('*[data-id="remixIdeSidePanel"]')
           .pause(3000)
           .perform((done) => {
-            plugins.forEach(plugin => {
+            const filtered = plugins.filter(plugin => plugin !== 'testremixIde') // remove this when localplugin bug is resolved
```

Comment:
This check can be done in the `forEach` statement below. No need to use `filter` for that.

Reply:
done